packages/block-library/src/image/edit.js

```diff
@@ -245,6 +246,16 @@ class ImageEdit extends Component {
 		};
 	}
 
+	getFilename( url ) {
+		if ( url ) {
+			const fileName = url.toString().match( /.*\/(.+?)$/ );
+			if ( fileName && fileName.length > 1 ) {
```

Comment:
Nitpick: If the regular expression doesn't match, fileName should just be null, so `if ( filename ) {` is sufficient.